packages/ui/src/molecules/PaymentMethods/PaymentMethods.tsx

```diff
@@ -0,0 +1,42 @@
+import type { ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface PaymentMethodsProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title of the payment methods section (e.g.: "Payment methods",
+   * "Accepted Cards").
+   */
+  title?: ReactNode
+  /**
+   * Children will receive the flags to be displayed in the payment
+   * methods section (e.g.:, visa, mastercard, etc).
+   */
+  children: ReactNode
+}
+
+const PaymentMethods = forwardRef<HTMLDivElement, PaymentMethodsProps>(
+  function PaymentMethods(
+    { testId = 'store-payment-methods', title, children },
+    ref
+  ) {
+    return (
+      <div ref={ref} data-store-payment-methods data-testid={testId}>
+        {!!title && <div id="payment-methods">{title}</div>}
```

Comment:
Thought for the future: we will need some function to auto-generate these IDs for us, because our clients may need to render the component more than once on the same page, causing the same ID to be used more than once when **they all should be unique per page**.

Reply:
Sure, it's something we need to discuss in the future.

Reply:
Sorry to chime on after merging. 🙈 
But maybe we could use `<span>` tag here instead of `<div>`, just because it's more usual for phrasing content, but doesn't have any semantic meaning either. (So it's okay to leave it as a `div`.
And I was wondering if we could use a list `ul` for the payment methods, but I'm not sure if it's always more than one payment method. 🤔 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span